README.md

```diff
@@ -956,18 +911,25 @@ end
 ```
 
 Replace the line `get "/", PageController, :index`
-with `live("/", Counter)`. So you end up with:
+with `live("/", Counter, layout: {LiveViewCounterWeb.LayoutView, "app.html"})`. So you end up with:
 
 ```elixir
 scope "/", LiveViewCounterWeb do
   pipe_through :browser
 
-  live("/", Counter)
+  live("/", Counter, layout: {LiveViewCounterWeb.LayoutView, "app.html"})
 end
 ```
 
-> 🏁 At the end of Step 12 you should have a `router.ex` file similar to:
-[`lib/live_view_counter_web/router.ex#L20`](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/008aaaca697015cc944bca6b99cc654b1385b51e/lib/live_view_counter_web/router.ex#L20)
+Here we're also telling LiveView how to define our root layout, `layout: {LiveViewCounterWeb.LayoutView, "app.html"}`. We need to do this since LiveView no longer uses the default app layout on the initial render of the application and needs to know which template file to use. Meaning that without these steps the application will never render! If you want to read more about the [`Live Layouts`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-live-layouts), the Hex documenation gives a good overview.
+
+The last thing we need to do for our root layout is use the `@inner_content` tag so we can have access to our render function that we defined in step 11.
+
+Open the `lib/live_view_counter_web/templates/layout/app.html.eex` again and make sure `<%= render(@view_module, @view_template, assigns) %>` is replaced with `<%= @inner_content %>` at the end of the `<main>` tag.
+
+> 🏁 At the end of Step 12 you should have your `router.ex` and `app.html.eex` files similar to:
+[`lib/live_view_counter_web/router.ex#L19`](lib/live_view_counter_web/router.ex#L19) and [`lib/live_view_counter_web/templates/layout/app.html.eex#L23`](lib/live_view_counter_web/templates/layout/app.html.eex#L23)
+
```

Comment:
Of course open to feedback here :) Used relative path links since some of these changes were made in this PR.